.eslintrc

```diff
@@ -1,2 +1,9 @@
-extends: jason
-parser: babel-eslint
+
+extends: "4catalyzer"
```

Comment:
my editor always just parses them as YAML automatically so i've starting just going with it, its nicer :P 

i'm not attached to it at all tho